vault/data_source_kv_secret_test.go

```diff
@@ -35,6 +39,10 @@ func TestDataSourceKVSecret(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, fmt.Sprintf("%s/%s-updated", mount, name)),
 					resource.TestCheckResourceAttr(resourceName, consts.FieldLeaseRenewable, "false"),
+					resource.TestCheckResourceAttr(resourceName, "data.zip", "zap"),
```

Comment:
Probably want to check the length of data.%

Reply:
Added in latest commit